app/main.py

```diff
@@ -1,25 +1,71 @@
 from abc import ABC
+from typing import Type, Any
 
 
 class IntegerRange:
-    pass
+    def __init__(self, min_amount: int, max_amount: int) -> None:
+        self.min_amount = min_amount
+        self.max_amount = max_amount
+
+    def __get__(self, instance: "Visitor", oven: "Visitor") -> None:
```

Comment:
Use `Any` for `instance` and `oven`, because you don't know type of these arguments
```suggestion
    def __get__(self, instance: Any, oven: Any) -> None:
```